src/translations/en/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "disclaimersCheckbox": "Disclaimers and Warnings (I know what I am doing)"
```

Comment:
Maybe re-word to "Hide disclaimers and warnings (I know what I am doing)"

I think this way it would be more clears what you are enabling - sorry, my last comment might have been worded badly.